draft-ietf-quic-transport.md

```diff
@@ -1000,16 +1003,16 @@ with an existing connection, or - for servers - potentially create a new
 connection.
 
 Hosts try to associate a packet with an existing connection. If the packet has a
-Destination Connection ID corresponding to an existing connection, QUIC
-processes that packet accordingly. Note that more than one connection ID can be
-associated with a connection; see {{connection-id}}.
+non-zero-length Destination Connection ID corresponding to an existing
+connection, QUIC processes that packet accordingly. Note that more than one
+connection ID can be associated with a connection; see {{connection-id}}.
 
-If the Destination Connection ID is zero length and the packet matches the
-address/port tuple of a connection where the host did not require connection
-IDs, QUIC processes the packet as part of that connection.  Endpoints SHOULD
-either reject connection attempts that use the same addresses as existing
-connections, or use a non-zero-length Destination Connection ID so that packets
-can be correctly attributed to connections.
+If the Destination Connection ID is zero-length and the packet matches the
+local address and port of a connection where the host used zero-length
+connection IDs, QUIC processes the packet as part of that connection.
+Endpoints that share a local address and port across multiple connections MUST
```

Comment:
I think that we can drop this final sentence here.  The text above (L933-ish) will do.

Reply:
Agreed, done.